static/docs/command-reference/get.md

```diff
@@ -48,7 +48,10 @@ configured in the project, containing the actual data.)
 > See `dvc get-url` to download data from other supported URLs.
 
 After running this command successfully, the data found in the `url` `path` is
-created in the current working directory, with its original file name.
+created in the current working directory, with its original file name. 
+Furthermore, if the target repository is local and, doesn't have a default 
+remote set up, dvc will try to get the cache from the target repository's cache 
+directory. 
```

Comment:
Here's my suggestion for this p:

After running this command successfully, the data found in the `url`/`path` combination is created in the current working directory, with its original file name. Note that if the target repository is local and doesn't have a default remote set up, DVC will try to get the cache from the target repository's cache directory.

Reply:
But actually the correctness of this will depend on the outcome of the question in https://github.com/iterative/dvc/pull/2915#pullrequestreview-332983712:

> does this logic mean that if there's a default remote setup, still DVC skips the local cache? Because I think it should try the local cache first regardless of remotes.

Reply:
Actually there's redundancy in this doc already. I will fix.